aries-backchannels/backchannel_operations.csv

```diff
@@ -87,11 +87,16 @@ rfc,command,response,topic,method,operation,id,data,description,id_details,data_
 0011 Revocation,X,,revocation-registry,GET,,Y,,Get revocation info from issuers webhook ,thread_id,,revocation_registry
 0036 Issue Credential,X,,credential,GET,,Y,,Get a stored credential from the holders wallet ,credential_id,,credential
 0036 Issue Credential,X,,credential,GET,revoked,Y,,Get a credentials revoked status ,credential_id,,revoked
+0434 Out of Band,X,,out-of-band,POST,send-invitation-message,,Y,Create an OOB invitation ,,invitation,invitation
+0434 Out of Band,X,,out-of-band,POST,receive-invitation,,Y,Receive invitation,,invitation,"state, connection_id"
```

Comment:
We'll need to add an "out of band" message for "reuse" eventually.  Not yet in OOB yet, but coming soon.